### DISCUSSION TITLE
Sometimes RDP gets stuck at the login screen and keeps spinning

### DISCUSSION THREAD

**Discussion by VShawn** [1Remote/1Remote#602] [Q&A]
**Describe the bug**

This issue occurs when using `1RM` `mremoteNG`, but I can successfully login using MSTSC.exe at the same time. 

RDP to other servers using `1RM` `mremoteNG` was working fine at that time.

After restarting the server, `1RM` `mremoteNG` working smoothly again. 

the reason is unknown for now.

**To Reproduce**

client: win10 22H2 
server: win10 22H2
LAN

Don't know, it happens occasionally.

**Screenshots**
![Snipaste_2023-11-20_11-40-52](https://github.com/1Remote/1Remote/assets/10143738/995452af-4ec4-49e3-8962-367e4e7999bb)

![Snipaste_2023-11-20_11-40-18](https://github.com/1Remote/1Remote/assets/10143738/e5754188-8df4-4330-b556-943e75d5fb75)

**Comment by majkinetor** [comment]
Can confirm that this happened to me on 1 occasion.

**Comment by VShawn** [comment]
happened to me again with mstsc.exe, so it not a bug of 1Remote's exported AxMSTSCLib.dll
